### DISCUSSION TITLE
How can I revert to the compat layout?

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [ibhagwan/fzf-lua#1298] [Q&A] [answered]
I recently upgraded 30fb626..98f315b for some newer features, but I prefer the previous layout (it was simple and concise).

Basically, the previous layout looked something like this:

```
    (other results here)
▌   vstorage/src/sync/status.rs         
> status  < 1/94 (0) ───────────────────
```

The new layout is something like:

```
    (other results here)
▌   vstorage/src/sync/status.rs         
  :: <ctrl-g> to Disable .gitignore     
────────────────────────────────────────
~/s/g/~/vdirsyncer-rs/status            
```

I removed the shortcut hint with `header=false`, from https://github.com/ibhagwan/fzf-lua/issues/1195

However, I can't figure out how to remove:

- The horizontal line. I've no idea what to try for this one.
- The current working directory. I tried `prompt = ">"`, but that doesn't seem to work.

Any hints on tweaking these two?

**Comment by WhyNotHugo** [comment]
I forgot to include my code, it's currently:

```lua
local fzf_lua = require("fzf-lua")
fzf_lua.setup({
  winopts = {
    split = "belowright new",
    preview = {
      hidden = "hidden",
    },
  },
  fzf_opts = {
    ["--layout"] = "default",
  },
})
fzf_lua.register_ui_select()
```

And:

```lua
  require("fzf-lua").files({
    cmd = cmd,
    ignore_current_file = true,
    header = false,
  })
```

**Comment by WhyNotHugo** [comment, marked as answer]
I bisected this to figure out when it changed and it seems to be 8af3a5864b319883bb3c7f8da612890320b7f92f. Oddly, the commit message comments how the previous default was better UX, so I'm not sure why it was changed. It also explains how to switch back to the previous behaviour.

The full config to restore the compact/clean layout is:

```lua
  require("fzf-lua").files({
    cmd = cmd,
    ignore_current_file = true,
    header = false, -- Hide keyboard hints.
    cwd_prompt = false,
    fzf_opts = { ["--info"] = "inline" },
  })
```

**Reply by ibhagwan** [reply]
> Oddly, the commit message comments how the previous default was better UX, so I'm not sure why it was changed. It also explains how to switch back to the previous behaviour.

It mentions the loading indicator is better with full info, the reason for later change is two fold, fzf (the binary) also introduced `inline-right` which made it possible to have the separator line as well as info on the right which makes the UI look better IMHO and also very close to Telescope.
